libmamba/include/mamba/core/output.hpp

```diff
@@ -54,6 +55,11 @@ namespace mamba
             right,
         };
 
+        const std::unordered_map<alignment, const char*> alignmentMarkers = {
+            { alignment::left, "alignment_left" },
+            { alignment::right, "alignment_right" },
+        };
+
```

Comment:
Isn't a map overkill here?
What about `auto enum_name(alignment) -> std::string_view`?

Reply:
good idea; changed in 34da4450